specification/appendix/examples/metadata/adding_new_columns_example.md

```diff
@@ -0,0 +1,78 @@
+# Adding New Columns
+
+## Scenario
+
+ACME has decided add additional columns to their FOCUS data export. The new columns are x_awesome_column1, x_awesome_column2, and x_awesome_column3. The provider creates a new schema object to represent the new schema, this schema object has a unique SchemaId. The subsequent data exports that use the new schema include the new schema's id as a reference to their corresponding schema object.
+
+## Supplied Metadata
+
+## Location for the new schema object
+
+`/FOCUS/metadata/schemas/schema-23456-abcde-23456-abcde-23456.json`
+
+## Content for the new schema object
+
+```json
+ {
+  "SchemaId": "23456-abcde-23456-abcde-23456",
+  "FocusVersion": "1.0",
+  "CreationDate": "2024-02-02T12:01:03.083z",
```

Comment:
Should this require an update on provider version?

Reply:
@AWS-ZachErdman, no it does not, they are decoupled. One reason being is that the provider may enable their customer to include certain columns. The Providers Version would be the same but columns would be added or removed based on customer configuration. Should the provider add/remove a column as part of an update to their process to deliver the FOCUS data then they would need to update Provider Version. That being said provider version is not a required component of the metadata and is up to the provider if they want to include.

Reply:
Ahh got it - I didn't realize it wasn't required.